e2e/main_test.go

```diff
@@ -157,6 +158,7 @@ func retry(attempts int, delay time.Duration, f func() error) error {
 		}
 
 		if i < attempts-1 {
+			glog.Errorf("attempt %d: %v", i, err)
```

Comment:
I think this turned out too spammy. Maybe remove?

Reply:
Yes. Will remove.